apps/OboeTester/app/src/main/cpp/cpu/AudioWorkloadTest.h

```diff
@@ -25,8 +25,9 @@
 #include <iostream>
 #include <thread>
 #include <vector>
+#include <cerrno>
```

Comment:
There are already places using oboe error code. In that case, why not use oboe error code?